bin/generate-docs

```diff
@@ -127,9 +129,9 @@ fetchApis()
 The documentation in this directory is automatically generated from the API entries
 generated from the [Manifest Reference](${process.env.MANIFEST_URL}).
 Detailed documentation with description, payload, and result formats is available on
-[${process.env.BASE_URL}/docs](${process.env.BASE_URL}/docs).
+[${urls.docs('/')}](${urls.docs('/')}).
```

Comment:
You can use `/docs` here, or `/docs/reference` - the relative URL should work on any cluster where these docs are displayed.